.github/workflows/ci.yml

```diff
@@ -115,6 +115,33 @@ jobs:
         if: github.ref == 'refs/heads/master'
         run: docker-compose push api
 
+  # test-wasm:
+  #   name: "[WASM] Test"
+  #   runs-on: ubuntu-latest
+  #   steps:
+  #     - uses: actions/checkout@master
+  #     - name: Cache cargo registry
+  #       uses: actions/cache@v1
+  #       with:
+  #         path: ~/.cargo/registry
+  #         key: ${{ runner.os }}-cargo-registry-${{ hashFiles('Cargo.lock') }}
+  #     - name: Cache cargo index
+  #       uses: actions/cache@v1
+  #       with:
+  #         path: ~/.cargo/git
+  #         key: ${{ runner.os }}-cargo-index-${{ hashFiles('Cargo.lock') }}
+  #     - name: Cache cargo build
+  #       uses: actions/cache@v1
+  #       with:
+  #         path: target
+  #         key: ${{ runner.os }}-cargo-build-target-${{ hashFiles('Cargo.lock') }}
+  #     - name: Run test
+  #       uses: actions-rs/cargo@v1
+  #       with:
+  #         command: test
+  #         args: -p gdlk_wasm
```

Comment:
Can you just remove this? Since it's just a copy of the core test job, it won't be hard to re-create if we write tests